projects/plugins/jetpack/json-endpoints/jetpack/class.jetpack-json-api-plugins-endpoint.php

```diff
@@ -420,6 +433,25 @@ protected function validate_plugin( $plugin ) {
 		return true;
 	}
 
+	/**
+	 * Validates if scheduled updates are allowed based on the current plan.
+	 *
+	 * @return bool|WP_Error True if scheduled updates are allowed or not provided, WP_Error otherwise.
+	 */
+	protected function validate_scheduled_updates() {
+		$args = $this->input();
+
+		if ( isset( $args['is_scheduled_updates'] ) && $args['is_scheduled_updates'] ) {
+			if ( Current_Plan::supports( 'scheduled-updates' ) ) {
+				$this->is_scheduled_updates = true;
+			} else {
+				return new WP_Error( 'unauthorized', __( 'This user is not authorized to perform a scheduled update event', 'jetpack' ), 403 );
```

Comment:
I guess a result of the site not having the Scheduled Updates feature is that users are not authorized to perform scheduled updates. From a user perspective reading this, the error message suggests it would need to be fixed by giving the user more permissions, when it's a matter of upgrading the plan (or purchasing an add-on if we decide to do that). How could this be phrased to be more helpful about next steps to take in order to fix this error?

Reply:
I've changed the text to "Scheduled updates are not available on your current plan. Please upgrade to a plan that supports scheduled updates to use this feature." I think this provides the reason for failing and also suggests the next steps here, what do you think?

Reply:
Love it!